apps/tasks.py

```diff
@@ -203,19 +203,14 @@ def deploy_resource(instance_pk, action="create"):
     app_instance = AppInstance.objects.select_for_update().get(pk=instance_pk)
     status = AppStatus(appinstance=app_instance)
 
-    if action == "create":
+    if (action == "create") or (action == "update"):
```

Comment:
Adding update action that actually updates the app parameters. This was not done before some some odd reason.